verifier.py

```diff
@@ -271,6 +271,24 @@ def US29_verify_deceased(individual):
 def US30_verify_living_married(individual):
     return individual['alive'] and individual['spouse'] != 'NA'
 
+# US33: List all orphaned children
+def US33_verify_orphans(individual):
+    parentsFamily = find_family(individual['child'])
+    father = find_individual(parentsFamily['husbandId'])
+    mother = find_individual(parentsFamily['wifeId'])
+    childAge =  individual['age']
+    if father['alive'] == False and mother['alive'] == False and childAge < 18:
+        return True
```

Comment:
Change this if-else to `return not father["alive"] and not mother["alive"] and childAge < 18`